velox/exec/CrossJoinProbe.cpp

```diff
@@ -116,6 +116,8 @@ RowVectorPtr CrossJoinProbe::getOutput() {
         rawIndices + (i + 1) * buildSize,
         probeRow_ + i);
   }
+  isIdentityProjection_ =
```

Comment:
Looks like it would be pretty rare when isIdentityProjection_ is true. Let's just never set it to avoid extra complexity.